closed/test/jtreg-ext/requires/OpenJ9PropsExt.java

```diff
@@ -38,6 +38,8 @@ public Map<String, String> call() {
         try {
             map.put("vm.bits", vmBits());
             map.put("vm.compiler2.enabled", "false");
+            map.put("vm.continuations", "false");
+            map.put("vm.gc.G1", "true");
```

Comment:
Setting G1 seems like it could cause problems later, even if it works today. Is it better to modify the test conditions to include tests we want for OpenJ9?

Reply:
Sure, I can test not set `G1` value, but remove `HumongousStack` test requirement for `G1` then.

Reply:
We could also add a new variable "vm.openj9" and modify the test condition to be `(vm.gc.G1 || vm.openj9`)?

Reply:
`vm.openj9` value has been added in the PR, test `HumongousStack` is running (though still fail with `VM does not support continuations`) when setting both values to `true` ([grinder_link](https://hyc-runtimes-jenkins.swg-devops.com/view/Test_grinder/job/Grinder/24806))